src/main/java/io/jenkins/plugins/analysis/core/steps/IssuesRecorder.java

```diff
@@ -494,7 +517,7 @@ private String createLoggerPrefix() {
     }
 
     private void record(final Run<?, ?> run, final FilePath workspace, final TaskListener listener,
-            final StageResultHandler statusHandler)
+                        final StageResultHandler statusHandler)
```

Comment:
is this formatting correct? Why did you change this line, it seems unrelated?

Reply:
Removed :+1: